.gitignore

```diff
@@ -119,3 +119,4 @@ wp-content/debug.log
 # add a line here that looks like this:
 # !/wp-content/themes/sirf
 !/wp-content/themes/sirf
+wp-content/themes/sirf/node_modules/*
```

Comment:
We should probably add this to the `.gitignore` in the sirf theme itself.

Reply:
Moved in https://github.com/INN/umbrella-sirf/pull/7/commits/5430fc9002c9cecf65ead323e69437ef9e58eecc